app/src/renderer/windows/account-manager/style.css

```
@@ -0,0 +1,618 @@
+  white-space: nowrap;
+}
+
+.account-manager__server-field,
+.account-manager__script-field {
+  height: var(--control-height-md);
+  min-width: 0;
+}
+
+.account-manager__script-field {
+  --script-actions-width: 6.75rem;
+  overflow: hidden;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)